src/node_file.cc

```diff
@@ -882,6 +888,62 @@ static void MKDir(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+
+static size_t CountSlashes(const char* str, size_t len) {
```

Comment:
Since I'm using a simple `for()`, `c_str()` would still need to be called. So figured I'd pull it out before.